src/shared/util/spawn.js

```diff
@@ -13,14 +13,18 @@ specific language governing permissions and limitations under the License.
 import cp from 'child_process';
 
 import colors from 'colour';
+import { IS_PACKAGED_BUILD } from '../build-info';
 
 export const spawn = (command, main, args, { logger }, options = {}) => new Promise((resolve, reject) => {
   logger.log('Spawning',
     colors.cyan(command),
     colors.blue(main),
     colors.yellow(args.join(' ')));
 
-  const child = cp.spawn(command, [main, ...args], { stdio: 'inherit', ...options });
+  const stdio = process.platform === 'win32' && IS_PACKAGED_BUILD
+    ? 'ignore'
+    : 'inherit';
```

Comment:
If there is a `.cmd` or `.exe` in the same location, then Windows seems to need to run it instead of the normal version most of the time. (`node` being an exception for some reason. Perhaps because it was installed by a Windows installer instead of being `npm install`ed?).  The `stdio` problem (as I understand it) is an `iojs` (or `nodejs`) limitation, not an `electron`-specific thing.

And I agree, it's really weird that this doesn't show up when running in non-packaged mode.